docs/application/tizen-studio/release-notes/3-6-release-notes.md

```diff
@@ -0,0 +1,59 @@
+- Web and Native IDE
+  - You can create unit tests for Tizen 2.3.2 and higher version projects only. Now Tizen Studio does not support unit testing for older versions.
+- Web IDE
+  - The preview tab in the Web Page Editor sometimes does not appear properly. Use an alternative feature, named Web SDK HTML Editor, which has enhanced features compared to the Web Page Editor. Instead of the preview tab in the Web Page Editor, use the preview feature **Ctrl + 4** of the Web SDK HTML Editor.
+  - In Rapid Development Support (RDS) mode, the web unit test result is not updated.
+- Certificate Manager
+  - Overwriting a duplicate certificate profile in the migration wizard works incorrectly on macOS.
+- Emulator
+  - In 5.5 Emulator Images app un-installation might take some time to complete if the app name contains more than 14 characters. A patch for the same will be released in next update.
+  - To use the Tizen emulator, install an Intel VTx supported by the CPU, and the latest version of the graphic card driver provided by the vendor. To verify the prerequisites for the Tizen emulator, see [Prerequisites for Tizen Studio](../setup/prerequisites.md).
+    - If the host machine is using NVIDIA&reg; Optimus&reg; technology on either Ubuntu or Windows&reg;, you must set the Tizen emulator to run with your NVIDIA&reg; graphics card. For Ubuntu, verify the [bumblebee project](https://wiki.ubuntu.com/Bumblebee). For Windows&reg;, select **High Speed NVIDIA&reg; Processor** as **Preferred Graphics Processor** in the NVIDIA&reg; control panel.
+    - On Ubuntu, if the graphics driver is out-of-date, your Ubuntu desktop session occasionally logs out while launching Emulator Manager, or the emulator skin is displayed improperly. Verify the prerequisites and upgrade to the latest graphics driver.
+  - On Windows&reg;, depending on your OS theme (such as Non-Aero themes and Windows XP themes), a display surface can be erased for a while if the emulator window is covered with another window. If you click the emulator window, the display surface runs correctly again.
+  - On Windows&reg;, if an error with message "failed to allocate memory" occurs while executing the emulator, try the following:
+    - Close some other programs and try to launch the emulator again.
+    - If the RAM size is set to 768 or 1024 MB for the VM in Emulator Manager, change it to 512 MB.
+    - Increase the user area of the virtual memory in the system to 3 GB by entering the **bcdedit /setincreaseuserva 3072** command on the console with administrator rights (only in Windows&reg; 7), and reboot.
+  - If you use a MacBook Pro that has both Intel HD and NVIDIA&reg; GPUs, the emulator can unexpectedly terminate when you execute the emulator with OpenGL ES version 1.1 or 2.0. Verify the emulator configuration in Emulator Manager and on the general tab in the emulator configuration window, set OpenGL ES version to version 2.0 or to version 3.0.
```

Comment:
... the emulator can be unexpectedly terminated ...

Reply:
done